react-multiple-components/my-app/src/RotatingBanner.tsx

```diff
@@ -0,0 +1,28 @@
+import Banner from './Banner';
+import NextButton from './NextButton';
+import PrevButton from './PrevButton';
+import Indicators from './Indicators';
+
+function RotatingBanner() {
+
+  const items = [
```

Comment:
For the next phase, this array should be defined in `App.tsx` and passed to `Rotating Banner` on props